process/block/preprocess/smartContractResults_test.go

```diff
@@ -434,12 +420,11 @@ func TestScrsPreprocessor_NewSmartContractResultPreprocessorNilEpochNotifier(t *
 		&testscommon.BlockSizeComputationStub{},
 		&testscommon.BalanceComputationStub{},
 		nil,
-		2,
 		&testscommon.ProcessedMiniBlocksTrackerStub{},
 	)
 
 	assert.Nil(t, txs)
-	assert.Equal(t, process.ErrNilEpochNotifier, err)
+	assert.Equal(t, process.ErrNilEnableEpochsHandler, err)
```

Comment:
rename to `TestScrsPreprocessor_NewSmartContractResultPreprocessorNilEpochsHandler`

Reply:
already renamed, looks more like a git issue